pxf/pxf-service/src/test/java/org/apache/hawq/pxf/service/rest/VersionResourceTest.java

```diff
@@ -36,7 +36,7 @@ public void getProtocolVersion() throws Exception {
 
         assertEquals(Response.Status.OK,
                 Response.Status.fromStatusCode(result.getStatus()));
-        assertEquals("PXF protocol version " + Version.PXF_PROTOCOL_VERSION,
+        assertEquals("{ \"version\": \"" + Version.PXF_PROTOCOL_VERSION + "\"}",
```

Comment:
Do you think an additional assert statement is required for MediaType to see if it is APPLICATION_JSON_TYPE?